examples/windows/z_pub_st.c

```diff
@@ -18,6 +18,8 @@
 #include <stdlib.h>
 #include <zenoh-pico.h>
 
+#define N 2147483647  // max int value by default
```

Comment:
It's safer to use `int n = ~0;` in such a way we always have the guarantee `n` is `max(int)` regardless the actual size of `int`.